apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -166,7 +166,7 @@ const InstructorsCell = withStyles(styles)((props: InstructorsCellProps) => {
                     </div>
                 );
             } else {
-                return profName;
+                return <div key={profName}> {profName} </div>;
```

Comment:
Incorrect! Never use `div`, `p`, or literally any native HTML tag in MUI...they have their own version for everything. I didn't choose this UI library, but that's just how the game is played :pensive: .

```suggestion
                return <Box key={profName}>{profName}</Box>;
```

On a technical note, the reason you would legitimately prefer `Box` over `div` is because it'll be styled properly according to whatever theme settings we applied for MUI, which results in greater visual consistency (supposedly, even though they're literally both `div` at the end of the day).

Since you're here, you could also replace all the other `div` tags with `Box` instead. It shouldn't really affect the UI at all, but it's just more aligned with the spirit of MUI :shrug: I already happened to do this, so you can use this as a reference (MUIv5) https://github.com/icssc/AntAlmanac/blob/aponia-monorepo/apps/client/src/components/School/Department/Course/Section/Section.tsx

Reply:
Box Box it is!